srcpkgs/NetworkManager-fortisslvpn/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'NetworkManager-openvpn'
+pkgname=NetworkManager-fortisslvpn
+version=1.3.90
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static"
+hostmakedepends="automake gettext-devel glib-devel intltool libtool pkg-config"
+makedepends="ppp-devel gtk+3-devel libglib-devel libnma-devel libsecret-devel"
+depends="openfortivpn"
+short_desc="NetworkManager VPN plugin for OpenFortiVPN"
+maintainer="Twilighttony <starrbeartony14@zoho.com>"
+license="GPL-2.0-or-later"
+homepage="https://wiki.gnome.org/Projects/NetworkManager"
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=884a7432bb38ae9c942e094f23d0a3feb504432153c3d2869ab71a4dd244d0fb
+system_accounts="_nm_openfortivpn"
+lib32disabled=yes
+_nm_openfortivpn_homedir="/var/empty"
+
+do_configure() {
+	./configure --prefix=/usr \
+	--sysconfdir=/etc \
+	--localstatedir=/var \
+	--libexecdir=/usr/lib/networkmanager \
+	--disable-static \
+	--with-pppd-plugin-dir=/usr/lib/pppd/$_pppver
+  make
+}
+
+post_extract() {
+	sed -i 's/\(#define NM_OPENFORTIVPN_\(USER\|GROUP\)\).*/\1 "_nm_openfortivpn"/' \
```

Comment:
You should probably use [vsed](https://github.com/void-linux/void-packages/blob/master/Manual.md#global-functions) for this.